php/class-customize-snapshot-command.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Customize Snapshot.
+ *
+ * @package CustomizeSnapshots
+ */
+
+namespace CustomizeSnapshots;
+
+/**
+ * Customize snapshot commands.
+ *
+ * @package CustomizeSnapshots
+ */
+class Customize_Snapshot_Command {
+
+	/**
+	 * Migrates snapshot posts into changesets.
+	 *
+	 * ## OPTIONS
+	 * [--dry-run]
+	 * : Check number of posts will effect via this process.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *      wp customize_snapshot migrate --dry-run
+	 *      wp customize_snapshot migrate
+	 *
+	 * @when after_wp_load
+	 *
+	 * @param array $arg        command args.
+	 * @param array $assoc_args assoc args.
+	 */
+	public function migrate( $arg, $assoc_args ) {
```

Comment:
I like to `unset( $arg )` for positional args that are intentionally not being used. This keeps static analysis happy.